CollAction/Views/Home/About.cshtml

```diff
@@ -139,3 +139,21 @@
     </div>
 
 </div>
+
+<script type="text/javascript">
+    var shiftWindowToCorrectForNavBarHeight = function ()
+    {
+        var navBarHeight = document.querySelector('.nav-row').offsetHeight;
```

Comment:
Yeah Ideally we should not have to use JS to change heights. I'm also not that keen in having too much javascript outside the frontend project. Is there a way to do this using CSS?